### ISSUE TITLE
Create PyHC logo and add it to website

### ISSUE THREAD

**Issue by namurphy** [heliophysicsPy/heliophysicsPy.github.io#94]
As far as I know, PyHC does not yet have a logo.  It would be great to create a PyHC logo so that we can add it to the website, use it in presentations, and make stickers to put on our laptops.  Ideally the logo could highlight the Sun and heliosphere, along with the interaction between the solar wind and planetary magnetic fields.

**Comment by sapols** [comment]
![A4C58704-7B7C-495E-951B-FA5C60CA53E9](https://user-images.githubusercontent.com/14846863/89793551-392e7a00-dae3-11ea-8a8c-b6ea6692f747.jpeg)

First draft.

**Comment by sapols** [comment]
I'm sorry that there hasn't been any update on this logo in a few weeks. I submitted change requests to the graphic designer three weeks ago, but they've been dealing with personal issues related to COVID and haven't been able to work on it.

**Comment by sapols** [comment]
![PyHC Logo w Text](https://user-images.githubusercontent.com/14846863/92411343-b1995280-f104-11ea-88aa-1399735526b6.png)

Second draft with improvements suggested from our telecon.

**Comment by sapols** [comment]
The font is just what we have on our website's homepage: Segoe UI. We could come up with an official PyHC font that's cooler if we want.

**Comment by sapols** [comment]
Second round of feedback: rotate Earth 90º (so that magnetic field is aligned with poles), add aurora to South Pole now that it's visible, add a sunspot or two, turn the sun's lines inward instead of outward.

**Comment by aburrell** [comment]
I'd also recommend changing the color of the aurora to make it easier to see and/or further stylizing the Earth by removing continents/covering them with simple clouds (again, making it easier to see the aurora).

**Comment by sapols** [comment]
Aaron suggests:
> The Sun should be at left, so the Earth’s filed can be turned 90deg to make it come from the poles.  A solar image instated of a ball would be nice too.

**Comment by BaptisteCecconi** [comment]
> Aaron suggests:
> 
> > The Sun should be at left, so the Earth’s filed can be turned 90deg to make it come from the poles.  A solar image instated of a ball would be nice too.

Although unusual, I like the proposed "alternative" view of the Sun-Earth system. It is bit similar (Sun and Earth) but still different (sideways) than usual, and can thus catch attention.

**Comment by sapols** [comment]
I'll have them make both versions (Sun in the foreground / Sun at the left) and I'll make a poll to see which people like more.

**Comment by balapoduval** [comment]
It would be nice to have some sunspots or other activities on the Sun to make it more exciting.

**Comment by aharonroberts** [comment]
The Earth can stay on top, but the magnetic field has to be in its proper orientation, and properly connected to at the aurora.  The aurora should be at both poles, and more prominent as suggested by aburrell.  How about a solar CME starting to lift off, in addition to activity on the disk?

**Comment by sapols** [comment]
Sorry for the long delay again. Our graphic designer didn't work on this for a while because of covid-related family issues. But we got back a few versions, and I've started a poll. Please go vote on your favorite version(s)!

https://pollunit.com/polls/a726kyqm1ois_hs1wvymeg

![pink PyHC Logo collage](https://user-images.githubusercontent.com/14846863/95515011-2c5ec300-097a-11eb-82ff-db472987bb5f.jpg)

**Comment by aburrell** [comment]
I'm abstaining, mainly because none of the field lines look correct, and they aren't visible (especially when compared to the sun spots).  The aurora is also still hard to see, and isn't correctly connected to the field lines.  The sun looks nice, though.

**Comment by aharonroberts** [comment]
I agree with Angeline, although I voted for the one that seemed to have the fewest problems.  We should get comments and iterate again.  The solar “field lines” are incorrect, so I took the  one without them.  The Earth field lines are all incorrect, as Angeline notes.  It would be nice to have a suggestion of a compressed sunward side field and an elongated night side, and connected properly to the aurora.

Aaron
